src/Build/BackEnd/BuildManager/BuildManager.cs

```diff
@@ -1134,7 +1139,8 @@ private void EndBuildTelemetry()
         {
             OpenTelemetryManager.Instance.DefaultActivitySource?
                 .StartActivity("Build")?
-                .WithTags(_buildTelemetry!)
+                .WithTags(_buildTelemetry)
+                .WithTags(_telemetryConsumingLogger?.WorkerNodeTelemetryData.AsActivityDataHolder())
```

Comment:
originally I did not intend for tasks/targets to be the same event as the basic info, but it seems fine this way too